main/signals.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.postgres.search import SearchVector
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from main.models import Post
+
+
+@receiver(post_save, sender=Post)
+def update_search_post(sender, instance, **kwargs):
+    Post.objects.filter(id=instance.id).update(
+        search_post=SearchVector("title") + SearchVector("body")
```

Comment:
I was thinking about searching both `title` and `body` fields or separate them but I think searching at both seems the best option, at least for now.

Reply:
I thought it would be best to search for `title` and `body` as this is what basically composes the whole article.

Reply:
Cool, of course! Do you think there should maybe be a space between them, so that the last word of the title and the first of the body do not merge into one? Or is it irrelevant? Eg:

```py
SearchVector("title") + " " + SearchVector("body")
```

Reply:
I guess it should not matter here, this is just updating the index with the `title` and `body` content.